src/components/TodoPage/TodoPage.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoMain } from '../TodoMain';
+import {
+  addPost,
+  getTodos,
+  updatePost,
+  USER_ID,
+  deletePost,
+} from '../../api/todos';
+import { TodoHeader } from '../TodoHeader';
+import { TodoFooter } from '../TodoFooter';
+import { ErrorNotification } from '../ErrorNotification';
+import { Filter } from '../../types/Enum';
```

Comment:
```suggestion
import { Filter } from '../../types/Filter';
```
Rename file with filter enum